llvm/test/Analysis/LoopAccessAnalysis/depend_diff_types.ll

```diff
@@ -262,3 +262,220 @@ loop:
 exit:
   ret void
 }
+
+define void @different_type_sizes_strided_accesses_independent(ptr %dst) {
```

Comment:
It might be worth adding a few brief comments + TODO for the cases we can optimize.